### ISSUE TITLE
CustomRPC settings: URL box should be on the top

### ISSUE THREAD

**Issue by wbt** [MetaMask/metamask-extension#5605]
**Describe the bug**
When connecting to a custom RPC, the first options shown, which occupy the screen real estate, are about currency conversion (or "current conversion;" see Issue #5603). However, this conversion is meaningless for private-network ETH.  When someone's going in to connect to a custom RPC, the most important element for the user to enter is the URL.  

**To Reproduce**
1. Install MetaMask and create an account or log in to an existing install.
2. Click on the MetaMask icon in the browser.
3. Click the Network selector (where it might say "Main Ethereum Network" in the upper right).
4. Click "Custom RPC" (the last item in the list). 
5. The network selector disappears and the UI changes to this:
![custom rpc editor](https://user-images.githubusercontent.com/563406/47506147-8291d480-d83d-11e8-97d7-75ebfef77760.PNG)
6. Notice that most of the screen real estate is occupied by currency conversion and language settings.  You have to scroll down further to access the "New RPC URL" box (using a scroll bar that is mostly hidden; that's issue #5604): 
![new rpc url box](https://user-images.githubusercontent.com/563406/47507652-8ecb6100-d840-11e8-964e-55368e1751e1.PNG)

**Expected behavior**
The "New RPC URL" box is the first thing you see when entering the Custom RPC UI. 

**Browser details (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome 70.0.3538.77 (Official Build) (64-bit) (cohort: 70_77_Win)
 - MetaMask Version: 4.16.0
 - New / Beta UI

**Comment by bdresser** [comment]
after https://github.com/MetaMask/metamask-extension/pull/5134 is merged, we'll make sure the `Custom RPC` option scrolls to the proper point in settings (@alextsg @danjm)

**Comment by whymarrh** [comment]
I think we can close this in favour of #5790, if you want to chime in over there. (Thoughts @bdresser?)

**Comment by wbt** [comment]
I have chimed in over there and if [this comment](https://github.com/MetaMask/metamask-extension/issues/5790#issuecomment-461838951) were implemented, that would address this Issue too.  However, I have seen too many cases in which A was closed as a duplicate in favor of/covered by the description of issue B, but then plans to address issue B changed in a way that left issue A unaddressed and closed complete.  I would strongly prefer to leave this Issue _open_ until the fix is actually in and the Expected Behavior is observed.
